config/metloom.yaml

```diff
@@ -0,0 +1,27 @@
+providers:
+  - id: METLOOM
+    collections: 
+      - id: SNOTEL
+        datetime: 2017-04-01/2021-04-23
+        aoi_file: config/aoi/demo_landcover.json
+        max_items: 3
+        outdir: data/snotel_land_cover_north_america
+        assets:
+          - SWE
```

Comment:
passing in `- all` should download all assets

Reply:
Good catch!